mail-filter/dcc/dcc-1.3.158.ebuild

```diff
@@ -14,37 +14,49 @@ SLOT="0"
 KEYWORDS="~alpha amd64 arm ~arm64 ~hppa ~ia64 ppc ppc64 sparc x86" #
```

Comment:
revbump and unstable, this is too invasive

Reply:
agreed, done